assets/scripts/lobby9.js

```diff
@@ -534,12 +534,16 @@ function teamLeaderSetup(phase) {
       $("#missionsBox").css("left", (widthOfRoom/2) + "px");
 }
 
-//set up the hover over missions box and highlight participating members
-
-
-
-
-
+//when hovering over missions box, highlight members participating in mission
+$("#missionsBox").hover(function(){
+    for (var i = 0; i < gameData.proposedTeam.length; i++) {
+		$('[usernameofplayer="' + gameData.proposedTeam[i] + '"]').addClass("highlight-avatar");
+	}
+    }, function(){
+    for (var i = 0; i < gameData.proposedTeam.length; i++) {
+		$('[usernameofplayer="' + gameData.proposedTeam[i] + '"]').removeClass("highlight-avatar");
```

Comment:
For some reason I can't find my earlier comment about the 'highlight-avatar' class. Could you use another class instead of 'highlight-avatar' please? At the moment the PICK button is enabled when the right number of highlight-avatar classes are found on the players so if I hovered over m2 when I was picking m4 (if the team sizes are the same) the PICK button might enable. If you could change it to another classname and give it the same properties that would be great!

Reply:
Sorry, I must have missed that comment the first time. I made a new css class called "highlight-participating" with the same properties as the "highlight-avatar" class and used that here instead. I added the changes in a new commit - if everything looks okay then I can squash them down into one commit.